lib/vrt.rb

```diff
@@ -86,6 +86,12 @@ def find_node(vrt_id:, preferred_version: nil, max_depth: 'variant', version: ni
     end
   end
 
+  # Convenience method to easily get the most current or preferred version of a vrt_id
+  def correct_vrt_id(vrt_id, version = current_version)
```

Comment:
Optional params in this repo seem to be mostly kwargs. I think that'd make sense here